Makefile

```diff
@@ -16,6 +17,14 @@ units = $(addprefix systemd/, bootupd.service bootupd.socket)
 all: $(units)
 	cargo build ${CARGO_ARGS}
 
+.PHONY: create-build-container
+create-build-container:
+	${CONTAINER_RUNTIME} build -t bootupd-build -f Dockerfile.build
+
+.PHONY: build-in-container
+build-in-container: create-build-container
+	${CONTAINER_RUNTIME} run -ti --rm -v .:/srv/bootupd:z localhost/bootupd-build make
```

Comment:
If a user is using `docker` instead of `podman`, I don't think the `localhost/bootupd-build` image name will work...though, I haven't used `docker` in ages, so maybe I am wrong

Reply:
It won't -- I'll put in an override for that too.